Package.swift

```diff
@@ -0,0 +1,27 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "QuizTrain",
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "QuizTrain",
+            targets: ["QuizTrain"]),
+    ],
+    targets: [
+        .target(
+            name: "QuizTrain",
+            dependencies: [],
+            path: "Sources"),
```

Comment:
You shouldn't need this any longer now that all of the sources files are in `Sources/[target name]`.

```suggestion
```